Assets/Scripts/Audio/AudioController.cs

```diff
@@ -0,0 +1,106 @@
+﻿using UnityEngine;
+using System.Collections;
+using System.Collections.Generic;
+
+
+public class AudioController : MonoBehaviour {
+
+    private float maxVolume;
+	private float fadeTime;
+
+	public bool hasPlayer;
+
+    private BoxCollider2D audioRangeCollider;
+    private AudioSource audioSource;
+    public List<GameObject> dolls = new List<GameObject>();
+
+    // Use this for initialization
+    void Start() {
+        audioRangeCollider = GetComponent<BoxCollider2D>();
+        audioSource = GetComponent<AudioSource>();
+		maxVolume = MechanicAudioManager.getInstance ().maxVolume;
```

Comment:
Yea was thinking that they are the same, and it will be redundant to have a lot of local variables